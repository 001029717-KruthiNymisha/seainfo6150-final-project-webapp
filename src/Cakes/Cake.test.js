import React from "react";
import { render } from "@testing-library/react";
import Cake from "./Cakes.jsx";

it("renders the Cake component correctly", () => {
  const { container } = render(
    <Cake
      Category="GlutenFreeCakes"
      bookId="006-Gluten-Free Coffee Cake"
    />
  );
  expect(container).toMatchSnapshot();
});