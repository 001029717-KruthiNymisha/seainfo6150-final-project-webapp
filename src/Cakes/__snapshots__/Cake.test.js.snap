// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the Cake component correctly 1`] = `
<div>
  <div>
    <center>
      Data is fetching
    </center>
  </div>
</div>
`;
